crates/router/src/core/payments/routing.rs

```diff
@@ -298,7 +298,7 @@ pub async fn perform_static_routing_v1(
         .await
         .change_context(errors::RoutingError::FallbackConfigFetchFailed)?;
         #[cfg(feature = "v2")]
-        let fallback_config = admin::BusinessProfileWrapper::new(business_profile.clone())
+        let fallback_config = admin::ProfileWrapper::new(business_profile.clone())
```

Comment:
```suggestion
        let fallback_config = admin::ProfileWrapper::new(profile.clone())
```

Reply:
Variable names changes we are taking up in the upcoming pull request